spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../../lib/takeaway.rb'
+
+describe "FEATURE SPEC" do
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+  describe 'takeaway features' do
+    it 'US1 - see a list of dishes with prices' do
+      takeaway = Takeaway.new(Menu.new)
+      expect(takeaway.show_menu).to eq ({ ribs: 3,
+      beef: 4,
+      rolls: 3,
+      chips: 2,
+      pies: 5})
+    end
+
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+    it 'US2 - selects some dishes' do
+      takeaway = Takeaway.new(Menu.new)
+      #pass in the order
+      expect(takeaway.select({beef: 2, rolls: 3}, 17)).to eq ({beef: 2, rolls: 3})
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.